shared/styles/index.native.js

```diff
@@ -104,6 +105,9 @@ const globalStyles = {
 
 const navBarHeight = OS === OS_ANDROID ? 60 : NavigatorNavigationBarStyles.General.TotalNavHeight
 const tabBarHeight = 48
+// FIXME: StatusBar.currentHeight returns undefined on iOS in RN 0.34
+const statusBarHeight = OS === OS_ANDROID ? StatusBar.currentHeight : 20
```

Comment:
Pretty sure that's only supported on the last couple versions of android FYI. hopefully this does the right thing in those cases